kube/objects_filter.go

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2019 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kube
+
+import (
+	// Load client-go authentication plugins
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+)
+// ObjectsFilter stores names of namespaces that needs to be included or excluded while running checks
```

Comment:
Nit: This struct could grow in the future to hold other kinds of filters, so I'd omit the mention of namespaces form this comment.

Reply:
I agree. Fixed comment.